person.rb

```diff
@@ -2,13 +2,14 @@
 
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
 
   def initialize(age, name = 'unknown', parent_permision = 'true')
     @id = Random.rand(1..500)
     @name = name
     @age = age
     @parent_permision = parent_permision
+    @rentals = []
     super()
   end
 
```

Comment:
- [ ] Please add a method for adding rentals inside the person class to make `student` and `teacher` classes inherit from it and be able to add rentals see this [example](`: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship`) in a previous lesson